feature-group-definitions/font-palette.yml

```
@@ -1,4 +1,5 @@
 name: font-palette and @font-palette-values
+description: The `font-palette` CSS property specifies one of the many palettes contained in a color font that browser should use for that font.
```

```suggestion
description: The `font-palette` CSS property selects a color palette from the font or set a custom palette.
```

---

I'm a bit puzzled b y the "set a custom palette", unsure how it differs from "selects a color palette".

Here's another attempt:

```suggestion
description: The `font-palette` CSS property selects a color palette from the font or creates a new palette by blending together two color palettes from the font.
```

---

I haven't tried writing demos for this, but my understand is that you can do at least 3 things:

- Use a palette from the font without changes by using `@font-palette-values` with `base-palette`
- Override some or all colors using `override-colors`
- Mix any two palettes using `palette-mix()`

@tursunova is that right?

---

Yes, that's right. I'm not sure about the phrasing, but maybe add something like: 

"...selects a color palette from the font or creates a new palette **by overriding colors from the palette or** by blending together two color palettes from the font"

---

Thank you @tursunova!

Since a palette from the font is always the base, perhaps we can describe this as a palette from the font with optional overrides. Bringing it all together:
```suggestion
description: The `font-palette` CSS property selects a color palette from the font, optionally overriding individual colors in the `@font-palette-values` at-rule.
```

---

There's also the question of `palette-mix()`. It shouldn't be mentioned in this feature because it was introduced for animations, and even though it can be used as `font-palette: palette-mix(--a, --b)` without animation, I think that's a niche thing best not mentioned.